src/app/my-skills/my-skills-dialog-component-trait.ts

```diff
@@ -0,0 +1,12 @@
+import { StepDescription } from './step-description';
+
+export abstract class MySkillsDialogComponentTrait {
+
+  protected getLabelHint(description: StepDescription): string {
```

Comment:
I suppose this function will be used for resolving the hint messages of both the skill levels and the skill priorities. This is a great idea. I would make the name a bit more explicit, like `resolveStepsDescription`. What do you think?

Reply:
Yes, you are right 😊